### ISSUE TITLE
Update Glitch

### ISSUE THREAD

**Issue by cheme75** [mrlt8/docker-wyze-bridge#361]
Trying to get the wyze-bridge update, using cmd prompt as admin - getting this error:
C:\Program Files\Docker\Docker\resources\bin>docker-compose pull
[+] Running 0/1
 - wyze-bridge Pulling                                                                                             0.5s
error getting credentials - err: docker-credential-desktop resolves to executable in current directory (.\docker-credential-desktop.exe), out: ``

From the  docker desktop cli, I get this error:
# docker-compose pull
/bin/sh: 1: docker-compose: not found

any suggestions?

**Comment by mrlt8** [comment]
This seems like a windows issue? 
https://forums.docker.com/t/docker-credential-desktop-exe-executable-file-not-found-in-path-using-wsl2/100225/5

**Comment by cheme75** [comment]
> This seems like a windows issue? https://forums.docker.com/t/docker-credential-desktop-exe-executable-file-not-found-in-path-using-wsl2/100225/5

yup, that was it! all updated... also got doorbell rotation changed to only using ffmpeg. added my second v2 cam which is working with the "copy" vcodec, so all is coming together. Just became a grandfather today too! lots of good today!

**Comment by cheme75** [comment]
@mrlt8 - thought I had updated correctly, but at top of log it still shows 1.2.2 and I see 2 images in docker - with latest one from yesterday not running. Not sure what I did wrong???
![docker images](https://user-images.githubusercontent.com/16415255/166732319-741bf9a2-66d5-46fe-9b2c-d76724dbd0ed.PNG)

**Comment by mrlt8** [comment]
Hmm. Is that in the desktop app? 

Can you stop and delete the old container?

**Comment by cheme75** [comment]
> Hmm. Is that in the desktop app?
 Yes.
> Can you stop and delete the old container?
Wasn't sure if I should delete just the older or both and start over - but, yes, I deleted the older image, left the newer one, then ran> docker-compose -f docker-compose.yml up -d and now it is running the latest 1.4.1 - thank you once I again! 

I also added ENABLE_AUDIO_FRONT_DOOR=True snce I don't use audio on any others, will check it out shortly.
